### DISCUSSION TITLE
⏳ 🔗 Elementor Email Template Builder

### DISCUSSION THREAD

**Discussion by TheRealFreelancer** [elementor/elementor#25176] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I'm surprised that this would not have been considered years before. Still, of all possible ideas on how Elementor could extend its functionality, there should be one game-changing feature: Email Templates.

### Describe the solution you'd like.

Extend the functionality to build email templates for **all email templates** that are to be sent by WordPress (mainly WP core and WooCommerce, but also third party) and utilize the **Elementor Drag and Drop Builder to create the same level of creative designs as the website's pages with emails**.

The email templates would be part of the Elementor template system and override the default templates. Of course, they would only show those building elements (widgets) that work with HTML emails, and the HTML email code output would be optimized for that purpose, reflecting modern email standards.

### Describe alternatives you've considered.

There are alternative plugins, of course, but nothing I have tested so far (a lot) can replace a **Elementor integrated solution**.

### Additional context

This would fill a considerable gap and reduce one of the headaches people have with WordPress customization (many more holes to close in the future). But this one would be a game-changer.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nolderoos** [comment]
This would be an amazing angle for Elementor, especially because all the other options out there are either very heavily priced or lack customisation! +1
